scipy/sparse/base.py

```diff
@@ -100,7 +100,26 @@ def get_shape(self):
 
     shape = property(fget=get_shape, fset=set_shape)
 
-    def reshape(self, shape):
+    def reshape(self, shape, order='C'):
+        """
+        Gives a new shape to a sparse matrix without changing its data.
+
+        Parameters
+        ----------
+        shape : length-2 tuple of ints
+            The new shape should be compatible with the original shape.
+        order : 'C', optional
+            This argument is in the signature *solely* for NumPy
+            compatibility reasons for the time being.
```

Comment:
I don't see `order` ever being used. I'm not sure exactly how to signal that clearly in the docs, though. Maybe we can find some existing examples of this sort of thing to copy from?